### DISCUSSION TITLE
Font size in CQ map preview sometimes overlaps

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#2031] [Ideas / Feature Requests]
Hi, it would be nice if font size in CQ map preview is not fixed to 24 px.
In such way, in small displays, numbers are covering large areas, overlapping borders, etc...

Maybe instead of 24px to use 1em or 1.25 em, IMHO.

Line 156 in cqmap.js

Also, maybe to add full screen to OpenMaps script :)

Regards,
Dragan,  4O4A

**Comment by AndreasK79** [comment]
Hi Dragan, good suggestion. I find that 1em or 1.25em will be very small. 1.5em seems to be better. I can't really reproduce the problem here. Perhaps you could try to replace 24px with 1.5em and see if that fixes the overlap?

Full screen needs a plugin, as it is not built in. I'll take a look at it.

**Comment by 4O4A** [comment]
Hi Andreas,

Well, both 1.25 and 1.5 looks OK with me. Maybe it is better to have a smaller value for smaller displays (But, then, who needs to look at map at small display?!?). :)

**Reply by AndreasK79** [reply]
Changed to 1.5em in dev now. EM scales with different display sizes, px does not, so hopefully this will be better.

**Comment by AndreasK79** [comment]
@4O4A added full screen option now: https://github.com/magicbug/Cloudlog/pull/2039

You can try out the PR, or wait for it go be merged.

**Reply by 4O4A** [reply]
Works fine, Andreas. Thank you!
